WordTransformer/src/TransformerRuleLength3.java

```diff
@@ -0,0 +1,17 @@
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public class TransformerRuleLength3  {
+    //no fields no state
+//Predicate <Integer>p= i -> i>0;
+    //Predicate<String> check =input -> input.length()==3;
+
+    Predicate<String> check = input -> input.length()==3;
```

Comment:
You do not need this class. The check and the predicate are external to the transform method, so, should be a part of the either test or a main method where you test whether your code works.